Description: Disable doctests which rely too much on implied features
Forwarded: not-needed
Last-Update: 2024-10-10
--- a/src/lib.rs
+++ b/src/lib.rs
@@ -10,2 +10,3 @@
 #![cfg_attr(docsrs, feature(doc_auto_cfg, doc_cfg))]
+#![cfg(not(doctest))]
 // `rustdoc::broken_intra_doc_links` is checked on CI
